components/elm/src/dyn_subgrid/dynpftFileMod.F90

```diff
@@ -61,7 +63,7 @@ subroutine dynpft_init(bounds, dynpft_filename)
     character(len=*) , intent(in) :: dynpft_filename ! name of file containing transient pft information
     !
     ! !LOCAL VARIABLES:
-    integer  :: wtpatch_shape(2)                  ! shape of the wtpatch data
+    integer  :: wtpatch_shape(3)                  ! shape of the wtpatch data TKT
 
```

Comment:
To avoid multiple additional review comments: check all the modified files, and remove initials from the in-line comments.

Reply:
@thorntonpe I checked files and removed comment initials in the modified files. Thanks.